python/src/triton.cc

```diff
@@ -1859,9 +1859,9 @@ void init_triton_ir(py::module &&m) {
                  mlir::createTritonGPUAccelerateMatmulPass(computeCapability));
            })
       .def("add_tritonamdgpu_accelerate_matmul_pass",
-           [](mlir::PassManager &self, int tensorCoreVersion, int instrSize) {
+           [](mlir::PassManager &self, int tensorCoreVersion, int instrSize, bool supportWMMA) {
```

Comment:
I feel that we can also check if wmma is supported from tensorCoreVersion. Maybe 
if tensorCoreVersion == 4, then supportWMMA =True

Reply:
I'm not sure about this concern.
1. It will be a bit confusing when CDNA 4 is released
2. Client code will be using not proper parameters, because matrix cores and wmma are different features for different architectures